cyences_app_for_splunk/default/macros.conf

```diff
@@ -712,9 +712,9 @@ definition = ".ad.crossrealms.com, .crossrealms.com, .local"
 definition = ".ad.crossrealms.com, @crossrealms.com, @CROSSREALMS.LOCAL, .crossrealms.com, .local"
 # Put it with double quote on both side (compulsory)
 
-[cs_user_inventory_exclude_users]
-definition = NOT user IN ("$*")
-# To exclude the specific users, add it inside the brackets by comma separated
+[cs_user_inventory_exclude_data]
+definition = NOT (sourcetype="WinEventLog" Account_Domain="NT VIRTUAL MACHINE")
```

Comment:
We should apply this to environment and not commit.

For the repo, use the value * maybe.